k8s/deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: module-14
+spec:
+  selector:
+    matchLabels:
+      app: module-14
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: module-14
+    spec:
+      containers:
+        - name: todo-app
+          image: todo-app:latest
+          imagePullPolicy: Never
+          ports:
+          - containerPort: 5000
+          env:
+          - name: FLASK_APP
+            value: todo_app/app
+          - name: LOG_LEVEL
+            value: DEBUG
+          - name: LOGGLY_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: logglytoken
+                key: LOGGLY_TOKEN
```

Comment:
Please note this now requires a Loggly account to be setup for running locally (which wasn't required before)